### PR TITLE
KOGITO-7484: Knative CLI Workflow Plugin II

### PR DESCRIPTION
**Pull request by ljmotta**
## JIRA
https://issues.redhat.com/browse/KOGITO-7484

### Quarkus subcommand
Moved the previous `build`, `create` and `deploy` commands to the `quarkus` subcommand.
 - `kn workflow build`    -> `kn workflow quarkus build`
 - `kn workflow create`  -> `kn workflow quarkus create`
 - `kn workflow deploy` -> `kn workflow quarkus deploy`

#### create
It creates a new single-file project. The project contains a "Hello World" `workflow.sw.json` file.
`kn workflow create`

#### build
Build a container image and generate the "./kubernetes" output folder. An image name is required. It can be passed through the `--image` or `--image-name` flag.
`kn workflow build --image=my-image`

#### deploy
It deploys the project using the "./kubernetes" folder. The container image generated by the `build` command needs to be accessible by the Kubernetes cluster.
`kn workflow deploy`

#### dev
It builds a dev container image and starts a dev container using port 8080, which can be tweaked.
`kn workflow dev`

#### prune
`kn workflow prune`
The user can choose to wipe out all generated files.

### Other changes
- Bump up the Quarkus version to 2.13.0.Final
- Update the VS Code debugger for the single-file commands and the quarkus commands.

## Demo
https://www.youtube.com/watch?v=xDxK3kGXVKE

### CODE REVIEW

**Comment by ederign** [comment]
@ljmotta, I'll also update the docs and review this after back from my PTO. Week of 31 OCT

**Comment by domhanak** [comment]
@cimbalek fyi

File: packages/kn-plugin-workflow/pkg/command/build.go [hunk 1]
```diff
@@ -73,292 +85,175 @@ func NewBuildCommand() *cobra.Command {
...
-		Repository: viper.GetString("image-repository"),
-		ImageName:  viper.GetString("image-name"),
-		Tag:        viper.GetString("image-tag"),
-
-		Jib:       viper.GetBool("jib"),
-		JibPodman: viper.GetBool("jib-podman"),
-		Push:      viper.GetBool("push"),
-
-		Test: viper.GetBool("test"),
+		Registry:   viper.GetString("registry"),
```
Comment (cimbalek) [inline packages/kn-plugin-workflow/pkg/command/build.go:151]: This shouldn't be here as it was fix for https://issues.redhat.com/browse/KOGITO-7670

I think you mabe put this back during rebase.

File: packages/kn-plugin-workflow/pkg/command/build.go [hunk 2]
```diff
@@ -73,292 +85,175 @@ func NewBuildCommand() *cobra.Command {
...
+		Repository: viper.GetString("repository"),
+		ImageName:  viper.GetString("name"),
```
Comment (cimbalek) [inline packages/kn-plugin-workflow/pkg/command/build.go:153]: This shouldn't be here as it was fix for https://issues.redhat.com/browse/KOGITO-7670

I think you mabe put this back during rebase.

**Comment by ederign** [comment]
https://github.com/kiegroup/kie-tools/pull/1664

**Pull request closed without merging**